### ISSUE TITLE
Anonymous "_" argument not rendered consistently.

### ISSUE THREAD

**Issue by guenterrote** [kosmikus/lhs2tex#50]
The "_" that represents an anonymous argument is not converted consistently.

Here is the lhs input:
```
\documentclass{article}
%include polycode.fmt
\begin{document}

> t _ (0:_) = False

\end{document}
```
lhs2TeX converts the Haskell line as follows:
`\>[3]{}\Varid{t}\;\anonymous \;(\mathrm{0}\mathbin{:\char95 })\mathrel{=}\Conid{False}{}\<[E]%`
The first _ becomes `\anonymous`, the second one is `\char95`. They look different. How can I get a uniform appearance?

(In any case, `\mathbin{:}\char95` would be more logical.)

**Comment by kosmikus** [comment]
Yes, this is one of the various peculiarities which are known (at least to me) for a long time, and I've never fixed them because I did not unnecessarily want to break backwards compatibility.

lhs2TeX is often being used for papers describing all sorts of variants of Haskell-like languages, so sometimes having a bit more syntactic flexibility than in standard Haskell is actually welcome.

It is also confusing though. And at the very least, it would be good if we had a documented list of known deviations.
